batavia/types/Bytearray.js

```diff
@@ -197,6 +197,16 @@ Bytearray.prototype.__rmul__ = function(other) {
     return this.__mul__(other)
 }
 
+Bytearray.prototype.__rmod__ = function(other) {
+    let types = require('../types')
```

Comment:
Isn't this let/require a no-op?

Reply:
Importing types at the top level creates a circular dependency here, but in general I'm importing it just for NotImplementedType so I'll look at importing that at the top level as it comes from core, originally.

Reply:
I've swapped all these out to import the NotImplemented singleton from core and imported it at the top level for all usages.